From ee305b8f18e6a63d67b500c9bd610eaeb614a80a Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Tue, 23 Dec 2025 20:08:10 +0800
Subject: [PATCH 322/344] AOSCOS: Revert "FROMLIST: rust: generate a fatal
 error if BINDGEN_TARGET is undefined"

This reverts "FROMLIST: rust: generate a fatal error if BINDGEN_TARGET is
undefined" as it breaks build on architectures that do not HAVE_RUST.

Link: https://lore.kernel.org/all/6bd26802-e3ce-43e2-b28d-f23e13492458@aosc.io/
Signed-off-by: Mingcong Bai <jeffbai@aosc.io>
---
 scripts/Makefile.rust | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/scripts/Makefile.rust b/scripts/Makefile.rust
index ea17409d9bee5..0382ba8b7b266 100644
--- a/scripts/Makefile.rust
+++ b/scripts/Makefile.rust
@@ -11,8 +11,4 @@ BINDGEN_TARGET_um	:= $(BINDGEN_TARGET_$(SUBARCH))
 BINDGEN_TARGET_riscv	:= riscv64-linux-gnu
 BINDGEN_TARGET		:= $(BINDGEN_TARGET_$(SRCARCH))
 
-ifeq ($(BINDGEN_TARGET),)
-$(error add '--target=' option to scripts/Makefile.rust)
-else
 export BINDGEN_TARGET
-endif
-- 
2.52.0

